### PR TITLE
Default NULL handing for CARDINALITY function

### PR DESCRIPTION
**Pull request by lokax**
#5072

### CODE REVIEW

File: src/function/scalar/map/cardinality.cpp [hunk 1]
```diff
@@ -26,6 +30,9 @@ static void CardinalityFunction(DataChunk &args, ExpressionState &state, Vector
 
 static unique_ptr<FunctionData> CardinalityBind(ClientContext &context, ScalarFunction &bound_function,
                                                 vector<unique_ptr<Expression>> &arguments) {
+	if (arguments.size() != 1) {
+		throw BinderException("Cardinality must have exactly one arguments");
```
Comment (pdet) [inline src/function/scalar/map/cardinality.cpp:34]: should arguments be singular?

**Review by pdet — CHANGES_REQUESTED** [review]
Many thanks for the PR!

Could you add a test with a mix of valid and invalid maps?

File: src/function/scalar/map/cardinality.cpp (hunk 1 above)
Comment (lokax) [reply src/function/scalar/map/cardinality.cpp:34]: The function uses only one argument

**Comment by Mytherin** [comment]
Thanks!

**Pull request merged**